sections/main-product.liquid

```diff
@@ -68,52 +68,52 @@
   {% assign variant_images = product.images | where: 'attached_to_variant?', true | map: 'src' %}
```

Comment:
the formatting applied should be the right one from the shopify prettier/linter 🤷 

I didn't mean to change it specifically but I think it's meant to be that way to prevent long horizontal lines of text